diff -cNr schily-2020-11-25-org/sccs/sccs/cmd/src/diff.c schily-2020-11-25/sccs/sccs/cmd/src/diff.c
*** schily-2020-11-25-org/sccs/sccs/cmd/src/diff.c	2020-09-19 20:53:33.000000000 +0900
--- schily-2020-11-25/sccs/sccs/cmd/src/diff.c	2020-12-06 19:36:29.095289500 +0900
***************
*** 161,166 ****
--- 161,167 ----
  #include <schily/sysexits.h>
  #define	VMS_VFORK_OK
  #include <schily/vfork.h>
+ #include <schily/io.h>
  #include <schily/libport.h>
  
  #else	/* non-portable SunOS -only definitions BEGIN */